comms/dht/src/macros.rs

```diff
@@ -45,6 +45,7 @@ macro_rules! acquire_write_lock {
 }
 
 #[cfg(any(test, feature = "test-mocks"))]
+#[allow(unused_macros)]
```

Comment:
You _could_ just delete the unused macro :)

Reply:
Haha, it's not really unused and tests will not compile if I remove it. Though the obvious solution is to just not use it in the few places in tests it is used. I'll do that :)